netlink/net_ns.go

```diff
@@ -0,0 +1,28 @@
+//+build linux
+
+package netlink
+
+import (
+	"os"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// guessRootNetNSFd guesses the file descriptor of the root net NS
+// and returns 0 in case of failure
+func guessRootNetNSFd(procRoot string) int {
```

Comment:
`connectToRootNetNSFS` since we're not really guessing anymore?

Reply:
I went with `getGlobalNetNSFD`